OpenSim/Common/TRCFileAdapter.cpp

```diff
@@ -169,9 +167,15 @@ TRCFileAdapter::extendRead(const std::string& fileName) const {
     }
     
     const size_t expected{ column_labels.size() * 3 + 2 };
+    // Will read into in memory matrix, then create table from matrix
```

Comment:
"Will read into in memory matrix" was a little confusing to me. Consider "Will first store data in a SimTK::Matrix to avoid expensive calls to the table's appendRow()."